app/assets/stylesheets/css-conf.scss

```diff
@@ -55,3 +55,45 @@ $color_success: #2ca25f;
 .color_main {
 	background: $color_main;
 }
+
+/*
+ * Type scale
+ *
+ */
+$mega: 6rem;
+$submega: 5rem;
+$f1: 3rem;
+$f2: 2.25rem;
+$f3: 1.5rem;
+$f4: 1.25rem;
+$f5: 1.125rem;
+$f6: 1rem;
+$f7: 0.875rem;
+
+.f-mega {
+  font-size: $mega;
+}
+.f-submega {
+  font-size: $submega;
+}
+.f1 {
```

Comment:
It was an idea for prototyping quickly, but yes, we can delete it if it's messy.